repo/packages/B/beta-mke/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [
+    "*"
```

Comment:
There's nothing to upgrade from, so this should be safe - right?

Reply:
Yes, I would say we should be safe